app/src/main/java/org/schabi/newpipe/player/BackgroundPlayer.java

```diff
@@ -563,4 +574,37 @@ public void onCompleted() {
             lockManager.releaseWifiAndCpu();
         }
     }
+
+    public static class MediaButtonReceiver extends BroadcastReceiver {
+
+        public MediaButtonReceiver() {
+            super();
+        }
+
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            if (Intent.ACTION_MEDIA_BUTTON.equals(intent.getAction())) {
+                KeyEvent event = (KeyEvent) intent.getParcelableExtra(Intent.EXTRA_KEY_EVENT);
+                if (event.getAction() == KeyEvent.ACTION_UP) {
+                    int keycode = event.getKeyCode();
+                    PendingIntent pendingIntent = null;
+                    if (keycode == KeyEvent.KEYCODE_MEDIA_NEXT) {
```

Comment:
Might be a good idea to also add fast forward and rewind, since some media buttons support those when long-pressed.